### PR TITLE
Lily 0.1

### PR DESCRIPTION
**Pull request by little-lily-17**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -19,29 +20,43 @@ def hello_world():
...
+
 
 
 @app.route('/todo/<int:index>/<string:word>', methods=['PUT'])
 def update(index, word):
-    items[index] = word
-    return jsonify('items', items)
+    collection.find_one_and_update({"seq": index},
+                                 {"$set": {"data": word}})
+    return dumps(collection.find_one({"seq": index}))
```
Comment (pandiwaw) [inline app.py]: refactor this ugly function

File: app.py [hunk 2]
```diff
@@ -19,29 +20,43 @@ def hello_world():
...
 
 if __name__ == '__main__':
     app.run()
+
+
+class JSONEncoder(json.JSONEncoder):
+    def default(self, o):
+        if isinstance(o, ObjectId):
+            return str(o)
+        return json.JSONEncoder.default(self, o)
```
Comment (pandiwaw) [inline app.py]: not used, deleted.

**Review by pandiwaw — CHANGES_REQUESTED** [review]

**Comment by pandiwaw** [comment]
Don't forget to update README.md

File: app.py (hunk 2 above)
Comment (little-lily-17) [reply app.py]: done

File: app.py (hunk 1 above)
Comment (little-lily-17) [reply app.py]: done

**Review by pandiwaw — APPROVED** [review]

**Pull request merged**